### PR TITLE
radarr Typo fixes

### PR DESCRIPTION
**Pull request by XristosTafarlis**
Fixed a minor typo in "Community Naming Suggestions, Movie Files"

### CODE REVIEW

File: radarr/settings.md [hunk 1]
```diff
@@ -130,11 +130,11 @@ Also, note that for each individual settings page, there are some options at the
 
 - Radarr v4.2.2.6489 or newer
 
-`{Movie CleanTitle} {(Release Year)} {imdb-{ImdbId}} {edition-{Edition Tags}} {[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels}]{MediaInfo AudioLanguages}[{Mediainfo VideoCodec}]{-Release Group}`
+`{Movie CleanTitle} {(Release Year)} {imdb-{ImdbId}} {edition-{Edition Tags}} {[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{MediaInfo AudioLanguages}[{Mediainfo VideoCodec}]{-Release Group}`
```
Comment (bakerboy448) [inline radarr/settings.md:133]: not a typo- this is intentional

File: radarr/settings.md [hunk 2]
```diff
@@ -130,11 +130,11 @@ Also, note that for each individual settings page, there are some options at the
...
 
 - Older Radarr Versions
 
-`{Movie CleanTitle} {(Release Year)} {Edition Tags} [imdb-{ImdbId}]{[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels}][{Mediainfo VideoCodec}]{-Release Group}`
+`{Movie CleanTitle} {(Release Year)} {Edition Tags} [imdb-{ImdbId}]{[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}[{Mediainfo VideoCodec}]{-Release Group}`
```
Comment (bakerboy448) [inline radarr/settings.md:137]: not a typo- this is intentional

**Review by bakerboy448 — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
may as well ensure naming aligns with trash and remove the `older radarr version` section

`{Movie CleanTitle} {(Release Year)} - {Edition Tags }{[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}`

default

`{Movie CleanTitle} {(Release Year)} {imdb-{ImdbId}} {edition-{Edition Tags}} {[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}`

Plex with edition support

Others:
https://trash-guides.info/Radarr/Radarr-recommended-naming-scheme/

**Comment by XristosTafarlis** [comment]
removed the older radarr version section

{Movie CleanTitle} {(Release Year)} - {Edition Tags }{[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}

And kept only new one

{Movie CleanTitle} {(Release Year)} {imdb-{ImdbId}} {edition-{Edition Tags}} {[Custom Formats]}{[Quality Full]}{[MediaInfo 3D]}{[MediaInfo VideoDynamicRangeType]}{[Mediainfo AudioCodec}{ Mediainfo AudioChannels]}{[Mediainfo VideoCodec]}{-Release Group}

**Review by bakerboy448 — APPROVED** [review]

**Pull request merged**